tests/ftest.sh

```diff
@@ -20,7 +20,7 @@ export PERF8_TRACE=${PERF8_TRACE:-False}
 export REFRESH_RATE="${REFRESH_RATE:-5}"
 export DATA_SIZE="${DATA_SIZE:-medium}"
 export RUNNING_FTEST=True
-export VERSION='8.16.0-SNAPSHOT'
+export VERSION='8.17.0-SNAPSHOT'
```

Comment:
I expect this is going to cause things to fail, until Elasticsearch produces an 8.17.0-SNAPSHOT. We need to find a way around this.

Reply:
Should we add the `skip-tests` label to be able to merge and avoid the chicken and egg problem?